README.md

```diff
@@ -0,0 +1,40 @@
+# Lansing Codes website
+
+[Lansing Codes](https://www.lansing.codes) is a website of resources for coders
+and other tech enthusiasts in and around Lansing, Michigan.
+
+These resources include a calendar of tech events, a list of meetup groups,
+local beginner-friendly resources, shout-outs of our gracious sponsors, and
+other ways to get involved in the Lansing tech community.
+
+## Code of Conduct
+All participants are expected to treat others with respect and follow our [Code of Conduct](https://github.com/lansingcodes/www/blob/master/.github/CODE_OF_CONDUCT.md).
+
+
+## Questions
+
+For general support, direct your questions to the
+[Lansing Codes Slack team](http://slack.lansing.codes). The issue list for this
+project is exclusively for bug reports and feature requests.
+
+## Stay in touch
+
+- [Slack](http://slack.lansing.codes)
+- [Twitter](https://twitter.com/lansingcodes)
+- [Facebook](https://www.facebook.com/lansingcodes)
+- [Website](https://www.lansing.codes)
+- [Newsletter](http://bit.ly/lansing-codes-newsletter)
+
+## Contribution
+
+You are welcome and encouraged to make changes to this website by submitting
+pull requests or forking our code to make your own community website!
+
+Before you get ahead of yourself, though, please read our
+[Contributing Guide](https://github.com/lansingcodes/www/blob/master/.github/CONTRIBUTING.md).
+
+## License
+
+[MIT - Hippocratic](https://firstdonoharm.dev)
+
+Copyright (c) 2015-present, Humanity Codes LLC
```

Comment:
Can you remove these last two sections? The instructions for contributing to this project and the license are yet to be determined so I think it's better to leave them out instead of putting placeholders.

Reply:
yes, removed both contributing and license sections